data/recData/style.css

```diff
@@ -0,0 +1,91 @@
+body {
+  width: 400px;
+  height: 320px; }
+
+h1 {
+  font-size: 1em;
+  font-weight: normal; }
+
+.addon-box {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 30%;
+  width: 400px;
+  border-top: gray solid 1px; }
+  .addon-box div {
+    display: flex;
+    flex-wrap: wrap;
+    align-content: center; }
+  .addon-box h2, .addon-box h3 {
+    width: 100%;
+    font-weight: normal;
+    margin: auto; }
+  .addon-box .image-box {
+    height: 100%;
+    width: 20%; }
+    .addon-box .image-box img {
+      display: block;
+      margin: auto; }
+  .addon-box .inner-box-1 {
+    height: 100%;
+    width: 60%; }
+  .addon-box .inner-box-2 {
+    margin: 0;
+    height: 100%;
+    width: 15%; }
+    .addon-box .inner-box-2 a {
+      width: 100%;
+      padding: auto;
+      text-decoration: none; }
```

Comment:
definitely put the closing bracket on the next line and i'm a fan of the blank line between selectors too.

+1 for alphabetizing